torch/_subclasses/fake_tensor.py

```diff
@@ -305,6 +308,12 @@ def from_tensor(t, fake_mode):
     def __repr__(self):
         return f"FakeTensor({self.fake_device}, {self.size()}, {self.dtype})"
 
+    def new_empty(self, shape, dtype=None):
+        return torch.empty(shape, dtype=dtype)
```

Comment:
This is not correct. If dtype is none need to use `self.dtype`. But I guess this is temporary until `new_empty` supports symbolic sizes?